Tests/AppKernel.php

```diff
@@ -158,4 +160,17 @@ public function process(ContainerBuilder $container)
 
         return $container;
     }
+
+    /**
+     * @throws Exception
+     */
+    public function __serialize(): array
```

Comment:
:thinking: why this change?

Reply:
Not entirely sure right now (bear in mind that I did this 3 months ago), but AFAICR, it had something to do with something failing under Symfony 6.1